src/tool/subcommands/api_cmd.rs

```diff
@@ -28,6 +28,12 @@ pub enum ApiCommands {
         /// Snapshot input paths. Supports `.car`, `.car.zst`, and `.forest.car.zst`.
         #[arg()]
         snapshot_files: Vec<PathBuf>,
+        /// Test filter
+        #[arg(long, default_value = "")]
+        filter: String,
+        /// Cancel test run on the first failure
+        #[arg(long)]
+        fail_fast: bool,
```

Comment:
I'd prefer `--no-fail-fast` to match `cargo test` and `cargo nextest`

Reply:
Clap has little support for `--no-*` flags. Let's punt this for another time.